packages/aws-cdk-lib/scripts/verify-imports-resolve-same.ts

```diff
@@ -84,7 +84,7 @@ async function compileAndResolve(fileName: string, contents: string, symbolName:
   }
 
   // Return the filename
-  const srcFile = sym.declarations?.[0].getSourceFile().fileName.replace(/.ts|.js|.d.ts/, '');
+  const srcFile = sym.declarations?.[0].getSourceFile().fileName.replace(/[.](ts|js|d\.ts)$/, '');
```

Comment:
Let's maybe also escape the first `.` here, instead of using a one-element class, since we do that later in the regex anyway, so it's better to be consistent...?

```suggestion
  const srcFile = sym.declarations?.[0].getSourceFile().fileName.replace(/\.(ts|js|d\.ts)$/, '');
```